lib/database/achievement.php

```diff
@@ -79,7 +79,7 @@ function getAchievementsList(
     }
 
     $query = "SELECT
-                    ach.ID, ach.Title AS AchievementTitle, ach.Description, ach.Points, ach.TrueRatio, ach.Author, ach.DateCreated, ach.DateModified, ach.BadgeName, ach.GameID,
+                    ach.ID, ach.Title AS Title, ach.Title AS AchievementTitle, ach.Description, ach.Points, ach.TrueRatio, ach.Author, ach.DateCreated, ach.DateModified, ach.BadgeName, ach.GameID,
```

Comment:
These changes could impact the public APIs as they mostly just dump query results. Duplicated data shouldn't matter, unless you care about the bandwidth.

Reply:
Made that change before making the avatar functions more robust, allowing for multiple variants of field names. That one in particular is a remnant of that. I was aware that those additional fields would show up in the web API but I don'T think it would be much of a problem. It's a public change nontheless. I'll revert that.

Reply:
Just realized that achievementAvatar() and renderAchievementCard() already can take both variants. Definitely reverting.